docusaurus/docs/plugins/plugins-getting-started.md

```diff
@@ -1,30 +1,20 @@
 # Getting Started
 
-## Use Case: Rancher Dashboard Development
+This guide will walk through creating a new extension from scratch.
 
-The goal is to not complicate the development process for Rancher. Although the code has moved around and even once we pull out code into the `pkg` folder, the development
-process for Rancher will be the same as it is.
+## Prerequisites
 
-You should be able to use `yarn dev` as before - the UI will build and run just as it did pre-2.6.5.
+You will need a newer version of nodejs installed.
 
-You should be able to make changes to code and the UI should hot-reload in the browser.
+You'll also need the yarn package manage installed, which can be done with `npm install -g yarn`.
 
-The development workflow for Rancher Dashboard remains unchanged.
+## Creating a Skeleton Application
 
-## Use Case: Develop a new UI
+In order to develop a new Extension, you need an application UI to host it in during development.
 
-This use case covers our need to build separate UIs for [Harvester](https://harvesterhci.io/), [Epinio](https://epinio.io/) etc - but wanting to leverage the core of the Rancher Dashboard UI.
+Rancher provides a helper to create a skeleton application for you.
 
-For this, we would use the `@rancher/shell` [NPM package](https://www.npmjs.com/package/@rancher/shell) and leverage it in a new UI project.
-
-We have 3 published NPM packages:
-
-- [`@rancher/shell`](https://www.npmjs.com/package/@rancher/shell) - the core shell UI that can be incorporated into a new UI to give core functionality
-- [`@rancher/create-app`](https://www.npmjs.com/package/@rancher/create-app) - a simple package that can be used with the `yarn create` command to create a new UI
-- [`@rancher/create-pkg`](https://www.npmjs.com/package/@rancher/create-pkg) - a simple package that can be used with the `yarn create` command to create a new UI Package
-
-
-Now we can go ahead and create a new UI.
+we can go ahead and create a new UI.
```

Comment:
Is this line needed? If so `we` --> `We`

Reply:
Removed lind